qcodes/__init__.py

```diff
@@ -17,8 +17,6 @@
 # instrument list and running monitor
 add_to_spyder_UMR_excludelist('qcodes')
 
-from qcodes.version import __version__
```

Comment:
why removing this ? :( i thought that it's a common thing to have the root of the package have `__version__`?

Reply:
Look at the whole file :) It's there twice

Reply:
phew :) :) :) indeed, twice is too many ;)